### PR TITLE
#1044 Schema changes for submenus

### PR DESCRIPTION
**Pull request by CarlosNZ**
~~Implement sub-menus for Data Views. Started with Data Views as its the simplest. Once we're happy with the general way its working I'll include for other menus too.~~

Just makes small changes to schema to include submenu info.

Test with front-end https://github.com/openmsupply/conforma-web-app/pull/1515

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -74,7 +74,7 @@
     "adm-zip": "^0.5.5",
     "archiver": "^5.3.1",
     "archiver-zip-encrypted": "^1.0.11",
-    "bcrypt": "^5.0.0",
+    "bcrypt": "^5.1.0",
```
Comment (CarlosNZ) [inline package.json:77]: There was a problem with 5.0.0, but can't remember what it was...

(Unrelated to this issue though)

File: plugins/action_send_notification/src/sendNotification.ts [hunk 2]
```diff
@@ -1,5 +1,3 @@
-const USE_MAIL_HOG = false // Change to true to test with local Mailhog
```
Comment (CarlosNZ) [inline plugins/action_send_notification/src/sendNotification.ts:1]: Not used (in `config.ts` now)

File: src/components/actions/helpers.ts [hunk 3]
```diff
@@ -32,7 +32,7 @@ export const swapOutAliasedAction = async (templateId: number, action: ActionInT
 
   const aliasedAction = await DBConnect.getSingleTemplateAction(templateId, code)
 
-  if (!aliasedAction) throw new Error('No Action matching alias')
+  if (!aliasedAction) throw new Error('No Action matching alias: ' + code)
```
Comment (CarlosNZ) [inline src/components/actions/helpers.ts:35]: Unrelated, but small change to give more helpful error message for Preview errors

(See https://github.com/openmsupply/conforma-server/issues/1048#issuecomment-1581706398)

**Review by fergie-nz — APPROVED** [review]
Looks good

**Pull request merged**